### DISCUSSION TITLE
The Win10 start menu cannot be opened, and there is no response.

### DISCUSSION THREAD

**Discussion by Oct1AtJoe** [valinet/ExplorerPatcher#1665] [Q&A] [answered]
The Win10 start menu cannot be opened, and there is no response. The latest version of the software is my system version: 22621.1848. Please give me some guidance
![微信图片_20230725143109](https://github.com/valinet/ExplorerPatcher/assets/140482860/1c0cf06c-2b0c-4fa1-95d6-e2375d8712d7)

**Comment by pyrates999** [comment, marked as answer]
Try the following:

1. uninstall explorer patcher
2. delete the registry key: "Computer\HKEY_CURRENT_USER\Software\ExplorerPatcher"
3. reboot
4. Install the july update: https://support.microsoft.com/en-au/topic/july-11-2023-kb5028185-os-build-22621-1992-605fa18f-bd49-41d8-80b1-245080e26c3d
5. reboot when it prompts you to
6. do not install any update labeled as preview as those are not supported by explorer patcher
7. install explorer patcher
8. verify you get a notification that symbols have been downloaded
9. restart windows explorer using either the task manager or in the properties window of explorer patcher
10. reboot

Then apply your settings again in explorer patcher to see if it fixes it.

**Reply by Oct1AtJoe** [reply]
The console log output still does not have the symbol download log, as shown in the figure above. However, after reinstalling, I can use the start menu of Win10, which is amazing

**Reply by pyrates999** [reply]
Please mark it as answered if it was the answer.
